### PR TITLE
use prettier to format md files

### PR DESCRIPTION
**Pull request by Jimexist**
# Which issue does this PR close?

Closes #.

 # Rationale for this change

# What changes are included in this PR?

- [prettier](https://prettier.io)
- [fd](https://github.com/sharkdp/fd)

```bash
fd -e md md ballista docs -x prettier --write
```

# Are there any user-facing changes?

### CODE REVIEW

**Review by msathis — APPROVED** [review]
Look good 👍  @Jimexist should we also add a .prettierrc file if we used custom settings & lint task during build? Maybe a separate PR at a later time is fine as well.

File: datafusion/docs/cli.md [hunk 1]
```diff
@@ -45,16 +45,23 @@ docker run -it -v $(your_data_location):/data datafusion-cli
...
+    datafusion-cli [FLAGS] [OPTIONS]
 
 FLAGS:
     -h, --help       Prints help information
+    -q, --quiet      Reduce printing other than the results and work quietly
     -V, --version    Prints version information
 
 OPTIONS:
-    -c, --batch-size <batch-size>    The batch size of each query, default value is 1048576
+    -c, --batch-size <batch-size>    The batch size of each query, or use DataFusion default
```
Comment (alamb) [inline datafusion/docs/cli.md:61]: 👍

**Review by alamb — APPROVED** [review]
I reviewed the rendered changes manually by looking at
https://github.com/Jimexist/arrow-datafusion/tree/update-docs-format

And they look good -- thanks @Jimexist

**Comment by Jimexist** [comment]
> I reviewed the rendered changes manually by looking at
> 
> https://github.com/Jimexist/arrow-datafusion/tree/update-docs-format
> 
> 
> 
> And they look good -- thanks @Jimexist 

Thanks I've been thinking of adding a non failing CI job so that future changes can be detected

**Comment by Jimexist** [comment]
> Look good 👍  @Jimexist should we also add a .prettierrc file if we used custom settings & lint task during build? Maybe a separate PR at a later time is fine as well.

The philosophy of prettier is to apply as little config as possible so if not necessary I can leave out the rc file

**Pull request merged**